src/middleware/records.middleware.js

```diff
@@ -0,0 +1,33 @@
+export const validateGroupId = (req, res, next) => {
+    const { groupId } = req.params;
+    if (isNaN(parseInt(groupId, 10))) {
+        return res.status(400).json({ message: 'URL 경로 그룹 ID가 유효하지 않습니다.' })
+    }
+    
+    next();
+};
+
+export const validateRecordBody = (req, res, next) => {
+    const { authorNickname, authorPassword, exercise_type, time, distance } = req.body;
+
+    if (!authorNickname || !authorPassword || !exercise_type || time === undefined || time === null) {
+        return res.status(400).json({ message: '닉네임, 비밀번호, 운동 종류와 시간은 필수 항목입니다.'});
+    }
+
+    const validExerciseTypes = ['RUNNING', 'CYCLE', 'SWIMMING'];
+    if (!validExerciseTypes.includes(exercise_type)) {
```

Comment:
```suggestion
    if (!validExerciseTypes.includes(exerciseType.toUpperCase())) {
```
`.toUpperCase()`는 입력값이 'run', 'Run'등 다양한 입력을 처리하는데 도움이 될 거 같습니다!